<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuWindow</class>
 <widget class="QMainWindow" name="menuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 68, 139);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="welcomeMessage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 68, 139);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Welcome, User</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(16, 84, 160);
border-color: rgb(0, 68, 139);</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sensorsButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 68, 139);
border-color: rgb(0, 68, 139);</string>
    </property>
    <property name="text">
     <string>Sensors</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="accountButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 68, 139);
border-color: rgb(0, 68, 139);</string>
    </property>
    <property name="text">
     <string>Account</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="logOutButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>620</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 68, 139);
border-color: rgb(0, 68, 139);</string>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>881</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="homePage"/>
    <widget class="QWidget" name="sensorsPage">
     <widget class="QPushButton" name="overviewButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 65, 119);
border-bottom-color: rgb(0, 65, 119);
font: 600 11pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Overview</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="activityButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 600 11pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Activity</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="reportsButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
font: 600 11pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Reports</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="adminButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
font: 600 11pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Admin</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidgetSensorsMenu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>851</width>
        <height>541</height>
       </rect>
      </property>
      <widget class="QWidget" name="overviewPage"/>
      <widget class="QWidget" name="activityPage"/>
      <widget class="QWidget" name="reportsPage"/>
      <widget class="QWidget" name="adminPage">
       <widget class="QLabel" name="systemRolesTitle">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 65, 119);
font: 700 14pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>System Roles</string>
        </property>
       </widget>
       <widget class="QLabel" name="currentRolesTitle">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 700 14pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Current Roles</string>
        </property>
       </widget>
       <widget class="QLabel" name="adminTitle">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>120</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 700 12pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Admin</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="accountPage">
     <widget class="QLabel" name="myAccountTitle">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 65, 119);
font: 700 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>MyAccount</string>
      </property>
     </widget>
     <widget class="QLabel" name="userDetailsTitle">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 700 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>User Details</string>
      </property>
     </widget>
     <widget class="QLabel" name="usernameTitle">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 700 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="username">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="roleTitle">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>250</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 700 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Role</string>
      </property>
     </widget>
     <widget class="QLabel" name="role">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>290</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="systemNameTitle">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 65, 119);
font: 700 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>System Name</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
